core/modules/wmgr/python/config.py

```diff
@@ -96,41 +97,60 @@ def __init__(self, appConfig):
 
         self._db = None
         self._dbPriv = None
+        self._dbProxy = None
 
     def dbEngine(self):
-        """ Return database engine """
-        kwargs = {}
-        kwargs['query'] = dict(local_infile=1)
+        """ Return database engine.
+
+        Standard Pool class causes delays in releasing connections, so we
+        disable connection pooling by using NullPool class. Another reason to
+        disable pool is that MySQL connections cannot be shared across many
+        threads.
+        """
+        if self._db is not None:
+            return self._db
```

Comment:
Should not be any, engine is a sort of a factory for connections, if the connection pool is disabled then there is no state which depends on MySQL.